account_bank_statement_import_online_qonto/models/online_bank_statement_provider_qonto.py

```diff
@@ -0,0 +1,122 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import requests
+import json
+from datetime import datetime
+import pytz
+import re
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+from odoo.addons.base.models.res_bank import sanitize_account_number
+
+QONTO_ENDPOINT = 'https://thirdparty.qonto.eu/v2'
```

Comment:
Is there a test endpoint or something like that? Like in Transferwise for tests on runbot

Reply:
non test endpoint